src/episervertemplating/modules/_protected/InstantTemplates/ClientResources/Scripts/TemplateContentTypesCommand.js

```diff
@@ -27,7 +29,8 @@
         //      The CSS class which represents the icon to be used in visual elements.
         iconClass: "epi-iconPackage epi-icon--inverted",
 
-        label: "New from Template",
+        label: customLocalization.episerver.cms.command.newtemplate != null ? 
```

Comment:
Can you create a helper method fort his? Something like =>

translate(customLocalization.episerver.cms.command.newtemplate, "New from template");

Reply:
Changed.